packages/rest/src/body-parsers/body-parser.text.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {RequestBodyParserOptions} from '../types';
+
+import {text} from 'body-parser';
+import {inject} from '@loopback/context';
+import {Request} from '../..';
+import {RestBindings} from '../keys';
+import {is} from 'type-is';
+
+import {RequestBody, BodyParser} from './types';
+import {
+  BodyParserWithCallback,
+  DEFAULT_LIMIT,
+  getParserOptions,
+  parseRequest,
+} from './helper';
+
+export class TextBodyParser implements BodyParser {
+  name = 'text';
+  private textParser: BodyParserWithCallback;
+
+  constructor(
+    @inject(RestBindings.REQUEST_BODY_PARSER_OPTIONS, {optional: true})
+    options: RequestBodyParserOptions = {},
+  ) {
+    const textOptions = Object.assign(
+      {type: 'text/*', limit: DEFAULT_LIMIT},
```

Comment:
Interesting. JsonParser is using `type: 'json'` but TextParse is using `type: 'text/*'`. Cannot we use `type: 'text'` too? I am not familiar with body-parser internals and may be wrong, but still wants to ask.